### ISSUE TITLE
Support for 32-bit jails on 64-bit hosts

### ISSUE THREAD

**Issue by assarbad** [iocage/iocage#292]
Hi,

according to the documentation 32-bit jails on 64-bit hosts should be doable and even relatively easy.

Please consider adding support for this in `fetch` and `create`.

Thanks!

**Comment by skarekrow** [comment]
Please search next time :)

This is a duplicate of:
https://github.com/iocage/iocage/issues/188

**Comment by assarbad** [comment]
I'm not sure how this is a duplicate.

I am talking about *32-bit jails* (i386 binaries) running on a *64-bit host* (OS *and* architecture 64-bit).

When reading over the other issue (#188) the person who opened it was apparently writing about **the host being i386**. In fact your own response implies this as well.

I am talking about being able to run 32-bit *and* 64-bit jails *in parallel* **on a 64-bit host**. And that means, by the way, that simply using `uname -m` or setting `ftpdir` won't help, because the current local folder (or dataset) hierarchy appears to use the release name only.

Also, I had filed this *in preparation* of sending a pull request eventually. I was still trying to familiarize myself with the code. But I guess I'll just script my own solution instead, maybe based on `iocage`, maybe not.

Apologies for the noise.

**Comment by skarekrow** [comment]
The issue remains the same, we will not be supporting anything but 64-bit jails. They already have 32-bit compatible libraries shipped with them, and neither of us wish to support pure 32-bit jails. The condescension wasn't necessary as I'm quite aware of our structure ;)

There also wasn't a mention that you would like to send a PR for this yourself in the original body of the issue. You are encouraged to familiarize yourself with our code as @pr1ntf stated (https://github.com/iocage/iocage/issues/291#issuecomment-161782031). We look forward to reviewing PR's with things that we find useful, but this issue is not one of them.

Good luck on your own script if you decide not to contribute to `iocage`.
